src/test/resources/features/Header-tests.feature

```diff
@@ -0,0 +1,12 @@
+# // Author: Camilla
+Feature: Test header elements and functionality
+
+  Scenario: Verify that the header Home link navigates you to the right location
+    Given User is on the page https://webshop-agil-testautomatiserare.netlify.app/
```

Comment:
Should we write url in given? or should we write like user is on TheShop page?

Reply:
don´t know if it matters?